toolbox-reference_5b7bdc1d0428631d7a8a2fcc.md

```diff
@@ -215,9 +215,14 @@ Examples:
     cache store gems-$SEMAPHORE_GIT_BRANCH-revision-$(checksum Gemfile.lock) vendor/bundle
 
 Archives file or directory specified by path and associates it with key.
-Any further changes of path after the store operation completes will not
+As `cache store` uses `tar` leading `/` is automatically removed from the path.
+Also, any further changes of path after the store operation completes will not
 be automatically propagated to cache. The command always passes (exits with 0).
 
+Allocated disk space for cache is currently limited to 9.6G. 
```

Comment:
I would rephrase it as:

The allocated disk space...

Question: is that cache space allocated _per user_ or _per organization_?

Reply:
Cache store is created per project basis and also disk size is allocated per project.

Reply:
I did not know that :) I think you should mention that information on the page.

Reply:
Sorry for the confusion, this part is stated in the description paragraph.

```
Cache is created on a per project basis and available in every pipeline job. All cache keys are scoped per project.
```

Reply:
My fault – I always edit for clarity but this time it was not necessary :)